schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INTEGER,
```

Comment:
- [OPTIONAL] As an enhancement, consider making the `id` column **not null** and **autoincremented** with `id int NOT NULL IDENTITY(1,1)`.
- [OPTIONAL] Marking the `id` column as a **primary key** would be a great addition for ensuring unique records! Consider adding `PRIMARY KEY (id)`.